snuba/web/query.py

```diff
@@ -235,7 +237,14 @@ def _format_storage_query_and_run(
     table_names = ",".join(sorted(visitor.get_tables()))
     with sentry_sdk.start_span(description="create_query", op="db") as span:
         formatted_query = format_query(clickhouse_query, request_settings)
-        span.set_data("query", formatted_query.structured())
+        span.set_data(
+            "query",
+            {
+                "Query": formatted_query.structured(),
+                "Size (Bytes)": _string_size_in_bytes(str(formatted_query)),
```

Comment:
I would do `formatted_query.get_sql()`, just to be consistent. Just a nit though.

Reply:
`formatted_query.get_sql()` returns a string, and if it's longer than ~500 characters it will get truncated in the UI. Should it be split over a list of multiple strings? Eg, `snuba_query_raw` in the description of #2041.

Reply:
```suggestion
                "Size (Bytes)": _string_size_in_bytes(formatted_query.get_sql()),
```
I was just suggesting this change, for when you calculate the string size.

Reply:
That makes sense, I thought you meant change `formatted_query.structured()` to `formatted_query.get_sql()`.